src/blockdata/script.rs

```diff
@@ -381,10 +393,17 @@ impl Script {
         Script::new_p2sh(&self.script_hash())
     }
 
+    /// Compute the P2WSH output corresponding to this witnessScript (aka the "witness redeem
+    /// script")
+    #[deprecated(since = "0.28.0", note = "use to_v0_p2wsh method instead")]
+    pub fn to_v0_wsh(&self) -> Script {
+        self.to_v0_p2wsh()
+    }
```

Comment:
In 7dc5554dcbbb8f3275fa78db2a5e100fca9e3706

Why add a new API that is being deprecated? This should be removed.

Reply:
Sorry, my bad. Will remove when will be updating the PR according to the results of other discussions here.